package.json

```diff
@@ -1,5 +1,32 @@
 {
+  "name": "istanbuljs",
+  "version": "1.0.0",
```

Comment:
oh, shoot! you are 100% right :p I was thinking you meant the packages within the repo -- because I'm bad at reading.